bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/things/VeluxProduct.java

```diff
@@ -217,7 +217,10 @@ public String toString() {
     // Class helper methods
 
     public String getProductUniqueIndex() {
-        return this.name.toString().concat("#").concat(this.typeId.toString());
+        if (!v2 || serialNumber.startsWith(VeluxProductSerialNo.UNKNOWN)) {
+            return name.toString();
```

Comment:
typeId isn't included here anymore?

Reply:
Correct. This is explicitly intended.

{ The `uniqueIndex` is an attribute that must explicitly identify the device **_in the hub_**. The hub knows nothing of OH type ids or other decorations that the binding or the user might choose might apply. The bug in the previous code was precisely because the original coder lost track of the difference between attributes in the hub, and potentially decorated attributes in the binding. This bug fix takes it firmly back goes back to the basics again. }